thetanuts/thetanuts/config.py

```diff
@@ -276,6 +283,16 @@ def validate_bid(
         except Exception:
             return {'errors': True, "messages": ["signature invalid"]}
 
+        bidding_token = w3.eth.contract(
+            w3.toChecksumAddress(vault.functions.COLLAT().call()),
+            abi=get_abi("ERC20"),
+        )
+
+        # Check for sufficient assets in wallet
+        assetBalance = bidding_token.functions.balanceOf(signer_wallet).call()
+        if assetBalance < sell_amount:
+            return {'errors': True, "messages": ["insufficient bidding token in wallet"]}
```

Comment:
`errors` is expected to be an `int`
([here the definition](https://github.com/tradeparadigm/sdks/blob/3d0d997427152f60036132d0875b8f5fe4d01925/sdk_commons/sdk_commons/config.py#L30-L32))

So should correspond to the length of the messages list, if any

I propose to change the last three lines as:

```
if assetBalance < sell_amount:
    return {'errors': 1, "messages": ["insufficient bidding token in wallet"]}
if not isValid:
    return {'errors': 1, "messages": ["???????? bid is invalid"]}
return {'errors': 0}
```

Reply:
ok! i rearranged it such that it more clearly identifies the 

        # check sufficient tokens first
        # check signature and parameters
        ....
        except Exception: # Catches EVM revert, when signature submitted is invalid
            return {'errors': 1, "messages": ["signature invalid"]}

        if isValid:
          return {'errors': 0}
        else:
          return {'errors': 1, "messages": ["signature valid, bid parameters invalid"]}